app/Base/index.tsx

```diff
@@ -154,6 +154,18 @@ function Base() {
         [alerts, addAlert, updateAlertContent, removeAlert],
     );
 
+    useEffect(() => {
```

Comment:
Any reason we have this on the Base component?

Reply:
I was trying to window scroll the page to the top.